cupy/cuda/compiler.py

```diff
@@ -174,10 +259,17 @@ def _compile_with_cache_nvrtc(source, options, arch, cache_dir,
                 mod.load(cubin)
                 return mod
 
-    ptx = compile_using_nvrtc(source, options, arch, name + '.cu')
-    ls = function.LinkState()
-    ls.add_ptr_data(ptx, u'cupy.ptx')
-    cubin = ls.complete()
+    if backend == 'nvrtc':
+        ptx = compile_using_nvrtc(source, options, arch, name + '.cu')
+        ls = function.LinkState()
+        ls.add_ptr_data(ptx, u'cupy.ptx')
```

Comment:
In Python3 it's equivalent.

```suggestion
        ls.add_ptr_data(ptx, 'cupy.ptx')
```